html/index.html

```diff
@@ -0,0 +1,332 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>What is my IP address? &mdash; {{ .Host }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="ifconfig. What is my IP address? &mdash; The best tool to find your own IP address, and information about it."
+    />
+    <link rel="canonical" href="https://ifconfig.co/" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/pure/1.0.0/pure-min.css"
+      integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/pure/1.0.0/grids-responsive-min.css"
+      integrity="sha384-b92sF+wDNTHrfEtRaYo+EpcA8FUyHOSXrdxKc9XB9kaaX1rSQAgMevW6cYeE5Bdv"
+      crossorigin="anonymous"
+    />
+
+    {{ template "script.html" . }} {{ template "styles.html" . }}
+  </head>
+  <body>
+    <div class="content">
+      <div class="pure-g gutters center">
+        <div class="pure-u-1 pure-u-md-2-3">
+          <div class="l-box">
+            <h1>{{ .Host }} — What is my IP address?</h1>
+            <p><code class="ip">{{ .IP }}</code></p>
+            <p>
+              The best tool to find your own IP address, and information about
+              it.
+            </p>
+          </div>
+        </div>
+        <div class="pure-u-1 pure-u-md-1-3">
+          {{ if .Sponsor }}
+          <div class="l-box leafcloud-placement">
+            <div class="pure-g">
+              <div class="pure-u pure-u-md-1">
+                <div class="leafcloud-logo">
+                  <a href="https://www.leaf.cloud?utm_source=ifconfig">
+                    {{ template "leafcloud-logo.svg" "." }}
+                  </a>
+                </div>
+              </div>
+              <div class="pure-u pure-u-md-1">
+                <p>
+                  This site is graciously hosted by<br />
+                  <a href="https://www.leaf.cloud?utm_source=ifconfig"
+                    >Leafcloud – The Truly Sustainable Cloud</a
+                  >
+                </p>
```

Comment:
not sure if html formatting like above is intentional? 🤔

Reply:
√